### ISSUE TITLE
new era begins...........

### ISSUE THREAD

**Issue by sa6ta6ni6c** [UnamSanctam/SilentCryptoMiner#2321]
TG: @octanepremium93
Write me "daddy" in tg and I will spare ur asses

Inco, its for you also :3

**Comment by kronxg** [comment]
go to sleep

**Comment by gabjohn3** [comment]
bro u only ddosing xD ur not too important a simple patch will fuck u HAHAHA

**Comment by sa6ta6ni6c** [comment]
BEST MINER https://zelenka.guru/threads/6716772/
BEST MINER https://zelenka.guru/threads/6716772/

**Comment by Alcinzal** [comment]
Sadly nobody is gonna use your miner if your way of advertising is through ddosing...
I can't even visit the website so I have no idea what kind of miner you are advertising, but if it costs money, then it's a very dumb idea advertising it in this repo, a literal **free** silent miner project.
However if your miner is free to use, then most people, if not all, will gather their brain cells and figure out that there is something sketchy about it. Either that it is a virus for whoever downloads it, or maybe it steals hashrate from someone that legitimately spreads.

Either way if someone is reading this and is, for whatever reason, thinking about buying or downloading the miner from the link above, **DO NOT**.

**Comment by gabjohn3** [comment]
Nobody will buy anything from this kiddies don't worry they are very hungry btw xD

**Comment by Alcinzal** [comment]
Yeah I hope so. But what exactly are they doing? From my understanding they are sending many requests to the api endpoint with a bunch of random data from many different IPs? And they are getting through cloudflare?

**Comment by gabjohn3** [comment]
its like the attack before, they exploint endpoint that all only need a update

**Comment by sa6ta6ni6c** [comment]
> Sadly nobody is gonna use your miner if your way of advertising is through ddosing...
> 
> I can't even visit the website so I have no idea what kind of miner you are advertising, but if it costs money, then it's a very dumb idea advertising it in this repo, a literal **free** silent miner project.
> 
> However if your miner is free to use, then most people, if not all, will gather their brain cells and figure out that there is something sketchy about it. Either that it is a virus for whoever downloads it, or maybe it steals hashrate from someone that legitimately spreads.
> 
> 
> 
> Either way if someone is reading this and is, for whatever reason, thinking about buying or downloading the miner from the link above, **DO NOT**.

Good, cus it's free, I get a % from the mining.
